.github/workflows/build.yaml

```
@@ -21,13 +21,30 @@ jobs:
               with:
                   python-version: '>=3.9.0 <4.0.0'
             - run: python3 -m pip install -r ci/cilib/requirements.txt
+
             - uses: docker/login-action@v2
               with:
                   registry: ${{ env.REGISTRY }}
                   username: ${{ github.actor }}
                   password: ${{ secrets.GITHUB_TOKEN }}
 
-            - run: ci/publish-docker --dev
+            - name: Set frontend version
+              run: echo "FRONTEND_VERSION=`ci/version app`" >> $GITHUB_ENV
```

Maybe you can replace this with jq:

```suggestion
              run: echo "FRONTEND_VERSION=$(jq '.version' package.json)" >> $GITHUB_ENV
```

That way you can remove all the python stuff

---

Oh yes, great suggestion ! Thanks